docs/src/content/pages/installation-next-js.mdoc

```diff
@@ -181,63 +181,63 @@ This `homepageData` will give us a JSON object with all the fields inside the `h
 
 Actually, there is only one field for now.
 
-Where you call the `reader` API in your app will depend on whether you are building your site in Next's `pages` or `app` directory.
+Where you call the `reader` API in your app will depend on whether you are building your site in Next.js' `pages` or `app` directory.
 
-### Data-fetching in the pages directory
+### Data-fetching in the app directory
 
-When building pages in Next's `pages` directory, getting the data from the `reader` API must be done in the `getStaticProps` function for staticly generated sites, or in `getServerSideProps` for apps running on a server.
+When building pages in Next.js' `app` directory, getting the data from the `reader` API can be done directly in a server-component.
 
 ```jsx
-// src/pages/index.tsx
+// src/app/page.tsx
 import { createReader } from '@keystatic/core/reader';
 import config from '../../keystatic.config';
-import { InferGetStaticPropsType } from 'next';
 
-export async function getStaticProps() {
-  // 1. Create a reader
-  const reader = createReader('', config);
+// 1. Create a reader
+const reader = createReader(process.cwd(), config);
 
+export default async function Index() {
   // 2. Read the "Homepage" singleton
   const homepageData = await reader.singletons.homepage.read();
 
-  // 3. Return the homepage data as props
-  return {
-    props: { homepageData },
-  };
-}
-
-export default function Index({
-  homepageData,
-}: InferGetStaticPropsType<typeof getStaticProps>) {
   return (
     <main>
       <h1>{homepageData?.headline}</h1>
     </main>
   );
 }
+
 ```
 
-### Data-fetching in the app directory
+### Data-fetching in the pages directory
 
-When building pages in Next's `app` directory, getting the data from the `reader` API can be done directly in a server-component.
+When building pages in Next.js' `pages` directory, getting the data from the `reader` API must be done in the `getStaticProps` function for staticly generated sites, or in `getServerSideProps` for apps running on a server.
 
 ```jsx
-// src/app/page.tsx
+// src/pages/index.tsx
 import { createReader } from '@keystatic/core/reader';
 import config from '../../keystatic.config';
+import { InferGetStaticPropsType } from 'next';
 
-export default async function Index() {
+export async function getStaticProps() {
   // 1. Create a reader
-  const reader = createReader('', config);
+  const reader = createReader(process.cwd(), config);
```

Comment:
Let's also move this `createReader` call outside of the function like in the other example